js/client-battle-tooltips.js

```diff
@@ -1152,6 +1165,47 @@ var BattleTooltips = (function () {
 			basePower = Math.floor(basePower * abilityBoost);
 			basePowerComment = this.makePercentageChangeText(abilityBoost, ability);
 		}
+
+		var allyActive = pokemon.side.active;
+		var auraBoosted = '';
+		var auraBroken = false;
+		if (move.category !== 'Status' && allyActive.length > 1) {
+			for (var i = 0; i < allyActive.length; i++) {
+				var ally = allyActive[i];
+				if (!ally || ally.fainted) continue;
+				if (!auraBoosted && (ally.ability === 'Fairy Aura' && move.type === 'Fairy') ||
+				    		    (ally.ability === 'Dark Aura' && move.type === 'Dark')) {
```

Comment:
We definitely don't support this indent+space thing.

Reply:
Also note that && has higher operator precedence than ||.